### DISCUSSION TITLE
Thanks for the speeed!

### DISCUSSION THREAD

**Discussion by garyross4** [mdbootstrap/TW-Elements#1603] [Kind words & general feedback]
Kudos on the modularity options you introduced.
I was looking for something like that in other UI systems.
I'm working on optimizing my latest project with your tools right now - I have to rewrite it from scratch, but I think the performance is worth it.

I think your kit is pretty unique in that way - I have seen lots of tools providing CSS optimization, but not many offer to reduce the weight of JS as much as Tailwind Elements. I installing components in modules is a great idea an every UI kit should do that.

**Comment by filipkappa** [comment]
Thanks for your kind words! Performance is one of our key indicators when building TE.
